lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -35,9 +35,7 @@ class EstimatedInputLatency extends Audit {
       name: 'estimated-input-latency',
       description: 'Estimated Input Latency',
       optimalValue: SCORING_POINT_OF_DIMINISHING_RETURNS.toLocaleString() + 'ms',
-      helpText: 'Primarily a measure of the main thread\'s activity. ' +
-      'There is a 90% probabililty a user would encounter input latency of this amount or less. ' +
-      '10% of touches can expect additional latency.',
+      helpText: 'If your app takes more than 100ms to respond to user input, users perceive the app as laggy. <a href="https://developers.google.com/web/tools/lighthouse/audits/estimated-input-latency" rel="noopener noreferrer" target="_blank">Learn more</a>.',
```

Comment:
I just rewrote it. It's a little longer, but I think crystal clear now

![image](https://cloud.githubusercontent.com/assets/4713486/20579266/a3628a14-b180-11e6-9063-d0e949317258.png)